lib/aggregate/aggregate_store.rb

```diff
@@ -119,7 +119,7 @@ def to_store
 
         # Optimization: only write out values if they are not nil, if there is no schema migration and
         # the default value is nil. (Schema migrations and defaults depend on writing the value.)
-        if respond_to?(:data_schema_version) || !aa.default.nil? || !agg_value.nil?
```

Comment:
@primiti the ticket body, you state that you thought we weren't writing default values. However this currently only doesn't write the default if the default is nil and the value is nil.

Reply:
Aha!  That could be the issue.  So the question is what breaks if we change this?  Can we try this and see if anything breaks in web?